templates/account/is-paying-disconnect-button.php

```diff
@@ -0,0 +1,53 @@
+<?php
+    /**
+     * @package     Freemius
+     * @copyright   Copyright (c) 2016, Freemius, Inc.
+     * @license     https://www.gnu.org/licenses/gpl-3.0.html GNU General Public License Version 3
+     * @since       2.5.0
+     */
+
+    if ( ! defined( 'ABSPATH' ) ) {
+        exit;
+    }
+
+    /**
+     * @var array $VARS
+     * @var Freemius $fs
+     */
+    $fs                     = $VARS['freemius'];
+    $is_active_subscription = $VARS['is_active_subscription'];
```

Comment:
As mentioned in my previous review, we need to remove this, and introduce `$is_license_available` here and make the check against that.